docs/notebooks/pet_motion_estimation.ipynb

```diff
@@ -1,20 +1,38 @@
 {
  "cells": [
   {
-   "cell_type": "code",
-   "execution_count": 1,
-   "id": "4111cc1b-5123-407a-8ec6-56d65bc94bc4",
    "metadata": {},
+   "cell_type": "markdown",
+   "source": [
+    "# Positron emission tomography (PET) data realignment example\n",
+    "\n",
+    "This example shows how to estimate the head motion of a PET dataset using\n",
+    "`NiFreeze`.\n",
+    "\n",
+    "The notebook uses the `sub-02` dataset that was generated synthetically from\n",
+    "a real PET dataset by adding random motion. The dataset\n",
+    "can be installed from [GIN G-node](https://gin.g-node.org/nipreps-data/tests-nifreeze):\n",
+    "\n",
+    "```\n",
+    "$ datalad install -g https://gin.g-node.org/nipreps-data/tests-nifreeze.git\n",
```

Comment:
Still not true as the data does not use `datalad` yet, but it will eventually (related to issue #112).